src/Middleware/HeaderPropagation/samples/HeaderPropagationSample/Startup.cs

```diff
@@ -49,6 +49,9 @@ public void ConfigureServices(IServiceCollection services)
                 .AddHttpClient("test")
                 .AddHeaderPropagation();
 
+            services
+                .AddHttpClient("another")
+                .AddHeaderPropagation(options => options.Headers.Add("X-BetaFeatures", "X-Experiments"));
```

Comment:
Do I also need more convoluted examples for all the possible scenarios we can handle now?
Or is it better to keep it nice and simple?

Reply:
No, I really think we should keep the examples simple and approachable.